src/farcasterd/runtime.rs

```diff
@@ -322,47 +311,47 @@ impl Runtime {
             }
 
             // Add progress in queues and forward to subscribed clients
-            BusMsg::Ctl(Ctl::Progress(..))
-            | BusMsg::Ctl(Ctl::Success(..))
-            | BusMsg::Ctl(Ctl::Failure(..)) => {
+            event @ (CtlMsg::Progress(..) | CtlMsg::Success(..) | CtlMsg::Failure(..)) => {
```

Comment:
I don't quite follow why this pattern binding change was necessary. Is it just to evade the use of `request.clone()`?

Reply:
Yes, the pattern is just used to group 3 types, but we discard their content anyway at that level so calling clone is a real waste.